sympy/series/limits.py

```diff
@@ -144,6 +146,12 @@ def doit(self, **hints):
         if z0.is_positive:
             e = e.rewrite([factorial, RisingFactorial], gamma)
 
+        # limits of fibonacci series when the variable approaches infinity
+        # can be conviniently calculated if fibonacci is rewritten in closed
+        # form in terms of Golden Ratio
+        if z0.is_infinite:
+            e = e.rewrite(fibonacci,GoldenRatio)
```

Comment:
Please add a space after comma.

This rewrite line could probably be placed after the factor_terms line below when `z0` has already been checked. (The comment may not be needed either.)